.gitignore

```diff
@@ -0,0 +1,2 @@
+*.swp
+vendor
```

Comment:
How so? It's in the .gitignore for `tendermint/tendermint` too. What's the alternative? put it in the global gitignore?